config/rootfs/debos/scripts/strip.sh

```diff
@@ -10,8 +10,7 @@ rm -rf /etc/localtime
 cp /usr/share/zoneinfo/Etc/UTC /etc/localtime
 
 
-UNNEEDED_PACKAGES=" libfdisk1"\
-" tzdata"\
+UNNEEDED_PACKAGES=" libfdisk1"
```

Comment:
What is the difference in ramdisk size?  Ideally this should be a parameter in rootfs-configs.yaml to remove it when it's not neeed.  Maybe we could run cros-ec tests with NFS instead in this case, as it has the full rootfs image before being stripped.